src/sdk/pynni/nni/evolution_tuner/evolution_tuner.py

```diff
@@ -175,7 +175,6 @@ def update_search_space(self, search_space):
         search_space : dict
         """
         self.searchspace_json = search_space
-        randint_to_quniform(self.searchspace_json)
```

Comment:
can we remove this line? @xuehui1991

Reply:
As I've stated in the commit message. `randint_to_(q)uniform` is no longer used in all the tuners who claims to have used it. But still needs a further check.